src/main/java/ch/njol/skript/bukkitutil/BukkitUtils.java

```diff
@@ -2,14 +2,35 @@
 
 import ch.njol.skript.Skript;
 import org.bukkit.Registry;
+import org.bukkit.inventory.EquipmentSlot;
 import org.bukkit.potion.PotionEffectType;
 import org.jetbrains.annotations.Nullable;
 
+import java.util.HashMap;
+import java.util.Map;
+
 /**
  * Utility class with methods pertaining to Bukkit API
  */
 public class BukkitUtils {
 
+	private static final Map<EquipmentSlot, Integer> BUKKIT_EQUIPMENT_INDICES = new HashMap<>();
+	private static final Map<Integer, EquipmentSlot> BUKKIT_EQUIPMENT_INDICES_REVERSED = new HashMap<>();
+
+	static {
+		BUKKIT_EQUIPMENT_INDICES.put(EquipmentSlot.FEET, 36);
+		BUKKIT_EQUIPMENT_INDICES.put(EquipmentSlot.LEGS, 37);
+		BUKKIT_EQUIPMENT_INDICES.put(EquipmentSlot.CHEST, 38);
+		BUKKIT_EQUIPMENT_INDICES.put(EquipmentSlot.HEAD, 39);
+		BUKKIT_EQUIPMENT_INDICES.put(EquipmentSlot.OFF_HAND, 40);
+
+		BUKKIT_EQUIPMENT_INDICES_REVERSED.put(36, EquipmentSlot.FEET);
+		BUKKIT_EQUIPMENT_INDICES_REVERSED.put(37, EquipmentSlot.LEGS);
+		BUKKIT_EQUIPMENT_INDICES_REVERSED.put(38, EquipmentSlot.CHEST);
+		BUKKIT_EQUIPMENT_INDICES_REVERSED.put(39, EquipmentSlot.HEAD);
+		BUKKIT_EQUIPMENT_INDICES_REVERSED.put(40, EquipmentSlot.OFF_HAND);
+	}
```

Comment:
sorry to put more work on you but since this is all nicely separated out from equipment slot now, it's a good opportunity to make it a BiMap instead

Reply:
Alright I'll get on this shortly, working on slot comparison pr reverts rn